site/src/_layouts/base.html

```diff
@@ -0,0 +1,81 @@
+---
+title: ""
+description: ""
+navigation:
+  - name: Installations
+    path: /
+    matchPath: /index
+  - name: Statistics
+    path: /statistics
+  - name: Integrations
+    path: /integrations
+  - name: Add-ons
+    path: /add-ons
+---
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>{% if title %}{{title}} | {% endif %}Home Assistant Analytics</title>
+    <meta property="og:title" content="{% if title %}{{title}} | {% endif %}Home Assistant Analytics" />
+    <meta property="og:site_name" content="Home Assistant Analytics" />
+    <meta property="og:url" content="https://analytics.home-assistant.io/" />
+    <meta property="og:type" content="website" />
+    <meta
+      property="og:description"
+      content="Summary of the collected Home Assistant usage statistics."
+    />
+    <meta
+      property="og:image"
+      content="https://brands.home-assistant.io/analytics/icon.png"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@home_assistant" />
+    <meta name="twitter:title" content="Home Assistant Analytics" />
+    <meta
+      name="twitter:description"
+      content="Summary of the collected Home Assistant usage statistics."
+    />
+    <meta
+      name="twitter:image"
+      content="https://brands.home-assistant.io/analytics/icon.png"
```

Comment:
For a future PR: Since you set `twitter:card` to `summary_large_image`, a square image is not good. It needs to be rectangular.

Reply:
We could just create a hardcoded social image (not dynamically based on current graphs)

Reply:
https://github.com/home-assistant/analytics.home-assistant.io/issues/639